VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,43 @@
+# Video 3: What is AsyncAPI
+ 
+Hello everyone! Welcome!
+		
+This video will help you understand what AsyncAPI is and how to continue learning about this Open-Source project. 
+
+In this video, we'll talk about:
+ - What is AsyncAPI
+ - How did AsyncAPI come about
+ - What AsyncAPI brings 
+ 
+ Ready? Let's get started!
+
+## Definition of AsyncAPI
+
+AsyncAPI is an open-source specification that aims to make working with EDA as easy as it is to work with REST APIs today.
+
+An important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs. This partnership allows investment in a dedicated department working full time on AsyncAPI spec, tools and community-building aspects.
+
+The strategy is to continue improving, implementing resources that make it more complete and solid. This has led to an agreement of joinment between AsyncAPI Initiative and the Linux Foundation in 2021. As a result AsyncAPI Initiative is fostered and hosted by a neutral foundation that guarantees the long-term sustainability and success of the project. 
+
+AsyncAPI Initiative currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs. Let us embrace reusability!
+
+2. Make the development experience super seamless, from idea to production. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members by ensuring it is a community-driven project running under open governance.
+
+ ## How did it come about
+Founded by Fran Mendez, AsyncAPI started as a side project at Hitch (API Changelog). He identified a lack of tools in the message-driven space and searched for a solution for standardized message-driven specification systems. 
+
+ ## What AsyncAPI brings
```

Comment:
this is more about the spec

but last sentence is about initiative

when I look at the script, I have a feeling we should start by saying:

There are 2 terms that require explanation, AsyncAPI and AsyncAPI Initiative

- section one, what is AsyncAPI
- section two, what is AsyncAPI Initiative

section one ends with mention: "AsyncAPI specification was created by Franito Bonito Mendez because identified a lack of tools in the message-driven space and searched for a solution for standardized message-driven specification systems. To maintain the spec he formed AsyncAPI initiative, and blabla let us explain what it is

section two explains initiative, and its goals


---
ya know what I mean?
basically, do not mix, and keep that flow 😄

Reply:
Yes, I understand what you mean. I agree with you. 

I have changed the structure of the text to clarify this. Let's see what you think 🙂